harness/determined/cli/master.py

```diff
@@ -11,54 +11,62 @@
 
 @authentication.required
 def config(args: Namespace) -> None:
-    if args.log:
-        if args.level and args.color:
-            if args.color.lower() == "on":
-                master_config = bindings.v1Config(
-                    log=bindings.v1LogConfig(level=args.level, color=True)
-                )
-            elif args.color.lower() == "off":
-                master_config = bindings.v1Config(
-                    log=bindings.v1LogConfig(level=args.level, color=False)
-                )
-            else:
-                raise CliError(
-                    "Error: Invalid value for --color provided: "
-                    + args.color
-                    + ". This optional parameter only accepts 'on' and 'off' values."
+    resp = bindings.get_GetMasterConfig(cli.setup_session(args)).config
+    if args.json:
+        determined.cli.render.print_json(resp)
+    else:
+        print(yaml.safe_dump(resp, default_flow_style=False))
+
+
+@authentication.required
+def set_master_config(args: Namespace) -> None:
+    if args.__dict__["log.level"] and args.__dict__["log.color"]:
```

Comment:
this logic will get very convoluted once we support more and more overridable configs. is there a way this can use the logic in `det experiment create --config`? it does a similar thing in `parse_config_overrides`.